### PR TITLE
Student cypress to playwright

### PR DESCRIPTION
**Pull request by camliu89**
Add the student-task and student-dashboard specs

### CODE REVIEW

File: tutor/specs/e2e/student-task.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    'neque viverra justo nec ultrices dui sapien eget mi proin sed libero enim sed faucibus turpis in eu mi bibendum neque egestas congue quisque egestas diam in arcu cursus euismod quis viverra nibh' + 
+    'cras pulvinar mattis nunc sed blandit libero volutpat sed cras ornare arcu dui vivamus arcu felis bibendum ut tristique et egestas quis ipsum suspendisse ultrices gravida dictum fusce ut placerat' +
+    'orci nulla pellentesque dignissim enim sit amet venenatis urna cursus eget nunc scelerisque viverra mauris in aliquam sem fringilla ut morbi tincidunt augue interdum velit euismod in pellentesque' +
+    'massa placerat duis ultricies lacus sed turpis tincidunt id aliquet risus feugiat in ante metus dictum at tempor commodo ullamcorper a lacus vestibulum sed arcu non odio euismod lacinia at quis' + 
+    'risus sed vulputate odio ut enim blandit volutpat maecenas olutpatv blandit aliquam etiam risus sed vulputate odio ut enim blandit volutpat maecenas olutpatv blandit aliquam etiam landit volutpat' +
+    'maecenas olutpatv blandit aliquam etiam maecenas olutpatv blandit aliquam etiam'
+
```
Comment (nathanstitt) [inline tutor/specs/e2e/student-task.e2e.ts:30]: the faker lib has a lorum method, we could use that here. `fake.lorum.words(500)` I think would do this

File: tutor/src/screens/task/step/exercise-free-response.js [hunk 2]
```diff
@@ -184,7 +184,7 @@ class FreeResponseInput extends React.Component {
                       <RevertButton size="lg" ux={ux} />
                       <AnswerButton
                           size="lg"
-                          data-test-id="submit-answer-btn"
+                          data-test-id={`${ux.isSubmitDisabled ? 'submit-answer-btn-disabled' : 'submit-answer-btn'}`}
```
Comment (nathanstitt) [inline tutor/src/screens/task/step/exercise-free-response.js:187]: hm, I see what you're attempting here, but it seems like it's changing the code a bit too much to make it easier to test.  Instead how about testing that the input is actually disabled like this:
```
expect(
    await page.isDisabled('testEl=submit-answer-btn')
).toBeTruthy()
```

https://playwright.dev/docs/api/class-page#pageisdisabledselector-options

**Review by nathanstitt — COMMENTED** [review]
looks good Alex but I think we both fixed the indentation and now there's a conflict

File: tutor/src/screens/task/step/exercise-free-response.js (hunk 2 above)
Comment (camliu89) [reply tutor/src/screens/task/step/exercise-free-response.js:187]: @nathanstitt I wanted to do that, but somehow `page.isDisabled` is not available. Maybe we have an old version of Playwright or Puppeteer?

**Comment by camliu89** [comment]
PR closed. New PR: #3511

**Pull request closed without merging**